### ISSUE TITLE
Optimization investigation

### ISSUE THREAD

**Issue by thomasboyt** [thomasboyt/bipp#31]
looks like `CSSPropertyOperations.setValueForStyles` is taking a LOOOOOT of CPU resources

**Comment by thomasboyt** [comment]
this comes down to `Chart.js:102` where the chart is given a transform

```
        <div style={{
          'position': 'absolute', 'top': 0, 'left': 0,
          'transform': transform, 'willChange': 'transform'
        }}>
```

very surprised by this taking so long...

**Comment by thomasboyt** [comment]
will probably need a new strategy beyond updating `transform` 60 times a minute... seems too slow...

**Comment by thomasboyt** [comment]
closed in favor of #32
